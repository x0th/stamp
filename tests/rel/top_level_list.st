[Object.type, Object.type, Object.type]
