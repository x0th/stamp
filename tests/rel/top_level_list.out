STDOUT:
[Object, Object, Object]
STDERR:
